db/migrate/20170925101702_create_restaurants.rb

```diff
@@ -0,0 +1,15 @@
+class CreateRestaurants < ActiveRecord::Migration[5.1]
+  def change
+    create_table :restaurants do |t|
+      t.string :name
+      t.text :cuisine
```

Comment:
this field could have probably made it with a `string` instead of `text`. `text` is a very big field and uses alot of space, so its better to save some space and use `string` when possible